src/calculateRentalCost.js

```diff
@@ -29,6 +29,15 @@
 
 function calculateRentalCost(days) {
   // write code here
+  if (days < 3) {
```

Comment:
2. [CHECKLIST] Don't use literals in expressions (HARDCODE). Instead use constants explaining this values.

Create some variables to make this code more understandable to others.

Reply:
Thank you